agent/reconciler/reconciler_test.go

```diff
@@ -41,14 +42,15 @@ var _ = Describe("Byohost Agent Tests", func() {
 		fakeCommandRunner = &cloudinitfakes.FakeICmdRunner{}
 		fakeFileWriter = &cloudinitfakes.FakeIFileWriter{}
 		fakeTemplateParser = &cloudinitfakes.FakeITemplateParser{}
+		fakeInstaller = &reconcilerfakes.FakeIK8sInstaller{}
 		recorder = record.NewFakeRecorder(32)
 		hostReconciler = &reconciler.HostReconciler{
-			Client:           k8sClient,
-			CmdRunner:        fakeCommandRunner,
-			FileWriter:       fakeFileWriter,
-			TemplateParser:   fakeTemplateParser,
-			Recorder:         recorder,
-			SkipInstallation: true,
+			Client:         k8sClient,
+			CmdRunner:      fakeCommandRunner,
+			FileWriter:     fakeFileWriter,
+			TemplateParser: fakeTemplateParser,
+			Recorder:       recorder,
+			K8sInstaller:   nil,
```

Comment:
Shouldn't this be set to `fakeInstaller`?

Reply:
By default, all tests were written with `skipinstallation: true`, setting the `k8sInstaller: nil` will keep the same behavior.